docs/using-rucio-client.md

```diff
@@ -212,75 +171,56 @@ that:
 - The token scope claim is no less than the minimum scope (e.g. 'openid profile') required by the Rucio Auth server (configured there in the rucio.cfg file).
 - same as above is true for the use of audience claim
 - token issuer is known to Rucio Authentication server
-- the identity of the token ("SUB=<user sub claim>, ISS=<issuer url>") is assigned to an existing Rucio account (pre-provisioned)
+- the identity of the token ("SUB=<user sub claim, ISS=<issuer url") is assigned to an existing Rucio account (pre-provisioned)
```

Comment:
This sentence should read as:

the identity of the token ("SUB=<user sub claim>, ISS=<issuer url>") is assigned to an existing Rucio account (pre-provisioned)